ui/app/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -264,16 +266,7 @@ function DateTimePicker(props: DateTimePickerProps) {
                       setTimePickerOpen(true);
                     },
                 inputProps: {
-                  onChange: handlePopupOnlyInputChange,
-                  value: asLocalizedDateTime(internalDate, localeCode, {
-                    hour12,
-                    hour: dateTimeFormatOptions?.hour || '2-digit',
-                    minute: dateTimeFormatOptions?.minute || '2-digit',
-                    // If the timezone control isn't displayed, the time displayed may
-                    // be misleading/unexpected to the user, so if timezone isn't displayed,
-                    // display timezone here.
```

Comment:
We need to address this somehow.

Reply:
what if we go back to setting the value with the `asLocalizedDateTime` but with the default format values (`2-digit` for hour and minute), and that way we keep the displaying of the timezone when needed?

btw, the showTimeZoneSelector prop is not currently being set to false in any usage

Reply:
I pushed the change I mentioned in the comment above, please let me know if not want we want so I can revert.